app/views/docs/command-line.phtml

```diff
@@ -66,6 +66,14 @@ brew install --HEAD appwrite</code></pre>
     <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite login</code></pre>
 </div>
 
+<div class="notice margin-bottom"> 
```

Comment:
Maybe we should also mention something around here about the `~/.appwrite/prefs.json` file that's generated from the login.

The pref.json looks like:

```json
{
    "endpoint": "https://localhost/v1",
    "selfSigned": true,
    "cookie": "a_session_console_legacy=eyJpZCI6IjYzNTMwOGExMDY2NGYyODg5NmYzIiwic2VjcmV0IjoiODkzMDk5OWJjOGRiNTcyZjA0MGNjYTYyNDE3MGJiNjNjNmU5ODZjNjE0OWNmMDU1ZDliNjdiMDUzODk1MWYyMzQ3MDBlYjRjNTI1OGE5NzVhZGE1NmQzZmFmNGZmN2MyNzRjZGQ0OTRhMDk5MjJkMmRjN2FkMDhkY2I2YzViOGQzNzczYzc3MDQ4YmYyNmQ2MGJlY2Q2YmQzZTYyMjdlZjJhMjJjNWMxZTU1MDIyNmMyM2VhOWUwM2Y1ZTVlMjQ4NzViNWY2OTMyYmU4OTcyYjNlYTI4MjkwMDg0NmU2OWQ4YTkxOTE0ZTc5YWFkNTI1YzkwNDNlMzcwZmMyZDM2MCJ9; expires=Wed, 25-Oct-2023 17:58:20 GMT; path=/; secure; httponly",
    "key": "434095337d41baf0dd11b3cf1fc5c42883f099b2259c28ddbbccc1ad4078844b984669c3d5bdc49ff0b8a79b3b922e5ac3d47e2337d33e8e1c71d03810edc633156e5894b311800a7d21196f41a8baf864946b357b3b5a871155213d7655bc7a4d2dda3c06b7bf405a67384d24d95fc19d43e48ddcddffae8fde788db51a52e6"
}
```

Reply:
????? Whaaaaaaattttt ????? This is new

Reply:
I'm going to go with not adding this. I feel like this is getting started guide is already super busy already and you'd never have to edit/view this file.